🔥 🔥 🔥 Add About, Rules, Schedule, Onsite Contest, and many things! (#4)
ia-toki/apio-2020-web: onsite.md

---
layout: single-page-md
title: APIO 2020 Onsite
key: onsite
---

### Registration for Onsite Contest

> * There will be only 10 that are invited and 6 participants for each countries. The committee will accept the confirmation of the country that want to do it onsite in First Come First Serve base.

Reviewer: ```suggestion
* There will be only 10 that are invited and 6 participants for each country. The committee will accept the confirmation of the country that wants to do it onsite in the First Come First Serve base.
```

> * There will be only 10 that are invited and 6 participants for each country. The committee will accept the confirmation of the country that wants to do it onsite in the First Come First Serve basis.

Reviewer 2: `s/10/10 countries`?

> * There will be only 10 that are invited and 6 participants for each country. The committee will accept the confirmation of the country that wants to do it onsite in the First Come First Serve basis.

Reviewer 2: `s/wants to do it onsite/wants to participate in the onsite contest`

* To register your country, please refer first to the general [registration page](registration).

> * In the confirmation phase, the leader need to state their confirmation to participate in onsite contest.

Reviewer: ```suggestion
* In the confirmation phase, the leader needs to state their confirmation to participate in the onsite contest.
```

> * In the confirmation phase, the leader needs to state their confirmation to participate in onsite contest.

Reviewer 2: `s/in onsite/in the onsite`

> * In the registration phase, the leader need to state the participants that will participate in the onsite contest.

Reviewer: ```suggestion
* In the registration phase, the leader needs to state the participants that will participate in the onsite contest.
```

### Onsite Contest Information
* The APIO 2020 onsite have the same rule as the [online contest format](rules#online-contest-format), with several differences.
* The participants of each invited country will take the contest in Jakarta, Indonesia.
* The contest will be held on Sunday, 10 May 2020 (UTC+7 08.00) for 5 hours.

> * Beside the contest day, there will be a time for excursion and visiting popular places in Jakarta.

Reviewer: ```suggestion
* Besides the contest day, there will be a time for excursion and visiting popular places in Jakarta.
```

* There are no differences between the onsite participants and the online participants. The winners will be decided equally as if the onsite participants are the online participants.

> * If onsite participants [win medals](rules#medals-distribution), they will be awarded physical medals in closing ceremony / awarding night.

Reviewer: ```suggestion
* If onsite participants [win medals](rules#medals-distribution), they will be awarded physical medals in the closing ceremony / awarding night.
```

### Onsite Contest Schedule

> Draft schedule, subject to change! For the general schedule, check [schedule page](schedule).

Reviewer: ```suggestion
Draft schedule, subject to change! For the general schedule, check [the schedule page](schedule).
```

> Draft schedule, subject to change! For the general schedule, check [the schedule page](schedule).

Reviewer 2: `s/Draft schedule, subject to change/Plesae note that the following is a draft schedule and subject to change.`

| Day | Time (in UTC+7) | Activity |
|-----|------|----------|
| Friday, 8 May 2020   |       | Arrival and check in |
| Saturday, 9 May 2020 | 09.00 | Opening ceremony |
|                      | 11.00 | Practice session |
|                      | 13.00 | Lunch |
|                      | 14.00 | Small excursion |
|                      | 18.00 | Dinner |
|                      | 20.00 | Free time |
| Sunday, 10 May 2020  | 08.00 | Onsite competition |
|                      | 13.00 | Lunch |
|                      | 14.00 | Excursion |

> |                      | 18.00 | Dinner and closing ceremony (awading) |

Reviewer: ```suggestion
|                      | 18.00 | Dinner and closing ceremony (awarding) |
```
